packages/nimble-components/src/table-column/menu-button/cell-view/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@microsoft/fast-element';
+import { controlSlimHeight } from '../../../theme-provider/design-tokens';
+
+export const styles = css`
+    :host {
+        align-self: center;
+        width: 100%;
+    }
+
+    nimble-menu-button {
+        height: ${controlSlimHeight};
+        width: 100%;
+    }
+
+    .value-label {
+        margin-right: auto;
+        overflow-x: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        height: 100%;
+        align-content: center;
```

Comment:
This `align-content: center` styling does not seem to be having any effect in chromatic, but I'm not sure why. The result is that the text is too close to the top of the menu button rather than being centered. The text alignment within the button looks correct on my machine in chromium, FF, and webkit. I'm open to suggestions of how to resolve the chromatic issue.

Reply:
@atmgrifter00 helped me work through the problem. It looks better in chromatic now. @jattasNI - can you double check that it looks correct in safari as well?

Reply:
Nice! Looks good to me!

<img width="911" alt="image" src="https://github.com/ni/nimble/assets/10500124/6aa99063-b3c3-4aaa-806b-32aad849e92b">
<img width="911" alt="image" src="https://github.com/ni/nimble/assets/10500124/4b9ce6c8-44ac-4b77-a62d-47a23d32b4fa">